pkg/envoy/accesslog_server.go

```diff
@@ -166,11 +167,14 @@ func (s *AccessLogServer) handleConn(ctx context.Context, conn *net.UnixConn) {
 			continue
 		}
 
-		flowdebug.Log(func() (*logrus.Entry, string) {
-			return log, fmt.Sprintf("%s: Access log message: %s", pblog.PolicyName, pblog.String())
```

Comment:
package scoped logrus logger `log` can be deleted once this and https://github.com/cilium/cilium/pull/37894 have been merged

Reply:
Rebased and removing logger variable in https://github.com/cilium/cilium/pull/37894